docs/examples/rx/01_simple_connect/index.js

```diff
@@ -0,0 +1,26 @@
+// Required imports
+import { zip } from 'rxjs';
+const { ApiRx } = require('@polkadot/api');
+const { WsProvider } = require('@polkadot/rpc-provider');
+
+function main () {
+  // Initialise the provider to connect to the local node
+  const provider = new WsProvider('ws://127.0.0.1:9944');
+
+  // Create the API and wait until ready
+  const api = await ApiRx.create(provider).toPromise();
+
+  // We're using RxJs 'zip()' combination operator to get the emitted values
+  // of multiple observables as an array
+  zip(
```

Comment:
Do you have a preference in whether to declare an observable const before subscribing to it or just chaining it?
The docs seem to favour the 'declaring a const' variant, I went for chaining because I just followed the 'If you're only using it once you don't need to declare a const for it' rule.

so:
```
const observable = zip(obs1, obs2, obs3);
observable.subscribe(val => console.log(val));
```
vs:
```
zip(obs1, obs2, obs3).subscribe(val => console.log(val));
```

Reply:
Personally I prefer the 2nd one, there's less code.

Reply:
yeah...that's the one I went for. But the more I'm looking at it the messier it seems, especially when you have some kind of generator functions in subscribe() and/or zip().

If it's a simple one-liner I'd probably stick to 2 but for more complex cases I think I'll switch to 1 for the sake of readability